src/operators/terminology_resolver.cpp

```diff
@@ -0,0 +1,180 @@
+
+#include "datatypes/pointcollection.h"
+#include "datatypes/linecollection.h"
+#include "datatypes/polygoncollection.h"
+
+#include "operators/operator.h"
+
+#include <json/json.h>
+#include "../util/terminology.h"
+
+/**
+ * Operator for resolving strings with the terminology service from gfbio: https://terminologies.gfbio.org/
+ *
+ * parameters:
+ *  - column: name of the textual column to resolve
```

Comment:
@jdroenner @ChristianBeilschmidt would `attribute_name` be better here? We always talk about attributes not columns, like e.g. `textual_attribute_filter` and not `textual_column_filter`

Reply:
yes, attribute appears to be a better name.